coq-htt.opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+version: "dev"
+
+maintainer: [
+  "Aleks Nanevski"
+  "Alex Gryzlov"
+  "Germán Andrés Delbianco"
+]
+authors: [
+  "Aleks Nanevski"
+  "Ilya Sergey"
+  "Germán Andrés Delbianco"
+]
+homepage: ["https://github.com/imdea-software/htt"]
```

Comment:
There's the project page http://software.imdea.org/~aleks/htt/, which could be specified as a home page.

Reply:
It doesn't seems to be updated frequently.

Reply:
Indeed it doesn't. It does provide some description of the system and the list of relevant publications though, which might give some idea of the system compared to the empty description in this repo.

Maybe that information can be referenced/moved into README.md (not necessarily in this PR of course).

Reply:
Right. We should better leave the pkg as it is, and then update the README (and replace @aleksnanevski  website with a project github' page). I will create an issue later and assign it to @clayrat